go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/robfig/cron/v3 v3.0.1
 	go.uber.org/zap v1.18.1
 	k8s.io/api v0.19.2
-	k8s.io/apimachinery v0.23.3
+	k8s.io/apimachinery v0.23.4
```

Comment:
I can't build the image without this changes.

Reply:
what is the error?

Reply:
#13 [go_builder 4/5] RUN mkdir -p build/_output/bin     && GOOS=linux GOARCH=amd64 CGO_ENABLED=0 GO_LDFLAGS=-w -s -trimpath         go build -mod=vendor -ldflags "-w -s -X main.GitCommit=7f2d8575a3fc8de00019bdbb090bc543831b7ae9 -X main.GitBranch=main -X main.BuildTime=2022-03-01T19:45:06Z"            -o build/_output/bin/percona-xtradb-cluster-operator                cmd/manager/main.go     && cp -r build/_output/bin/percona-xtradb-cluster-operator /usr/local/bin/percona-xtradb-cluster-operator
#13 sha256:b76bf0a19f58ca1b3897534a507a9d75442cd0e20bc9b9eeb2d06e7200cabdcc
#13 DONE 179.7s

#14 [go_builder 5/5] RUN go get k8s.io/apimachinery/pkg/util/sets     && go build -o build/_output/bin/peer-list cmd/peer-list/main.go     && cp -r build/_output/bin/peer-list /usr/local/bin/
#14 sha256:dc56837ad08e7068233a280954077dd9096fed99d16e7c594c4a389d470e24db
#14 14.01 go: downloading k8s.io/apimachinery v0.19.2
#14 23.41 go get: upgraded k8s.io/apimachinery v0.23.3 => v0.23.4
#14 23.52 go: inconsistent vendoring in /go/src/github.com/percona/percona-xtradb-cluster-operator:
#14 23.52 	k8s.io/apimachinery@v0.23.4: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#14 23.52 	k8s.io/apimachinery@v0.23.3: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
#14 23.52
#14 23.52 	To ignore the vendor directory, use -mod=readonly or -mod=mod.
#14 23.52 	To sync the vendor directory, run:
#14 23.52 		go mod vendor
#14 ERROR: executor failed running [/bin/sh -c go get k8s.io/apimachinery/pkg/util/sets     && go build -o build/_output/bin/peer-list cmd/peer-list/main.go     && cp -r build/_output/bin/peer-list /usr/local/bin/]: exit code: 1
------
 > [go_builder 5/5] RUN go get k8s.io/apimachinery/pkg/util/sets     && go build -o build/_output/bin/peer-list cmd/peer-list/main.go     && cp -r build/_output/bin/peer-list /usr/local/bin/:
------
executor failed running [/bin/sh -c go get k8s.io/apimachinery/pkg/util/sets     && go build -o build/_output/bin/peer-list cmd/peer-list/main.go     && cp -r build/_output/bin/peer-list /usr/local/bin/]: exit code: 1
You can see the same error in main branch